api/src/integration-test/groovy/com/aldoborrero/tinder/api/services/LoginSpec.groovy

```diff
@@ -23,8 +27,24 @@ class LoginSpec extends TinderAbstractSpec {
     @Override
     def setupTinderService() {
         interceptor = Spy(TestAuthTokenInterceptor)
-        
-        def tinder = Tinder.create(new Configuration.BaseConfiguration() {
+
+
+        TinderEndpoint endpoint = new TinderEndpoint();
+        endpoint.setUrl("http://localhost:65119");
+        endpoint.setName("Production");
+
+        TinderErrorHandler errorHandler = new TinderErrorHandler();
+
+        // We need to change the naming. I don't like it.
+        def tinderPicasso = Tinder.with()
```

Comment:
I think is better to remove with() method (as is empty and doesn't provide nothing useful more than semantics) and use it as normal `Builder` something like:

```
Tinder tinder = new Tinder.Builder().setEndpoint().setAuthTokenInterceptor().build()
```